uk.co.compucorp.civicrm.tasksassignments/sass/civitasks/partials/_dashboard.scss

```diff
@@ -109,6 +109,26 @@
 
                 .task-title {
                   margin: 4px 0;
+
+                  > .#{$prefix}task-title {
```

Comment:
*(Applies for similar selectors below)*

So this is equals to (stripping `#{prefix}` away for simplicity sake)
```css
.dashboard .dashboard-tasks--full-height-columns > main > .content-wrapper > .content .list_task > li header .task-title > .task-title {
    color: #707C88
}
```
That's a huge selector. Isn't there any way you can target that element via a shortest route?